rpm/Makefile

```diff
@@ -1,6 +1,6 @@
 PROJECT = distgen
 PYTHON  := python3
-VERSION := $(shell cd .. ; $(PYTHON) setup.py --version)
+VERSION := $(shell python3 -m distgen --version | grep -o '[0-9]*\.[0-9]*')
```

Comment:
`.github/workflows/copr/copr-custom-script`:
`python3 -m pip install . --user`

This change was done so that we can just grab the version directly from the package, use the up-to-date package to build it as an extra sanity test that everything works and prevent weird parsing of `toml` file in bash.

I think I would ideally not use `Makefiles` at all, but that is just my personal opinion and definitely out of scope for this change.